hardware/neopixel/neopixel.html

```diff
@@ -31,6 +31,16 @@
         <option value="grb">GRB</option>
     </select>
 </div>
+<div class="form-row">
+    <label for="node-input-gamma"> </label>
+    <label for="node-input-gamma" style="width:70%">
+    <input type="checkbox" id="node-input-gamma" style="display:inline-block; width:22px; vertical-align:baseline;"> Gamma corrected
+    </label>
+</div>
```

Comment:
Is there a few words that a layman can understand what this means ? (maybe not - but just asking)

Reply:
I'll change it to "Apply gamma correction"

Reply:
changed to "Apply gamma correction"